scipy/stats/morestats.py

```diff
@@ -3478,3 +3478,76 @@ def circstd(samples, high=2*pi, low=0, axis=None, nan_policy='propagate'):
         R = np.minimum(1, hypot(sin_mean, cos_mean))
 
     return ((high - low)/2.0/pi) * sqrt(-2*log(R))
+
+
+def quartile_coeff_dispersion(a, q=(0.25, 0.75), axis=0,
+                              interpolation='lower'):
+    """
+    Compute the quartile coefficient of dispersion for a given sample array.
+
+    Given two quartiles Q_low, Q_high from array samples such that
+    Q_low < Q_high, the quartile coefficient of dispersion is calculated by
+    (Q_high - Q_low)/(Q_high + Q_low).
+    by default it is computed using 3rd and 1st quartiles.
+
+    Parameters
+    ----------
+    a : array_like
+        Input array.
+    q : array_like of float, optional
+        Sequence of 2 quantiles, such that q[0] < q[1].
+        The default is to compute with Q_high = 0.75, Q_low = 0.25.
+    axis : {int, tuple of int, None}, optional
+        Axis along which the coefficient of dispersion is computed.
+        Default is 0. If None, compute over the whole array a.
+    interpolation : {'linear', 'lower', 'higher', 'midpoint', 'nearest'}
+        This optional parameter specifies the interpolation method to
+        use when the desired quantile lies between two data points
+        ``i < j``:
+
+        * linear: ``i + (j - i) * fraction``, where ``fraction``
+          is the fractional part of the index surrounded by ``i``
+          and ``j``.
+        * lower: ``i``.
+        * higher: ``j``.
+        * nearest: ``i`` or ``j``, whichever is nearest.
+        * midpoint: ``(i + j) / 2``.
+
+        The default is 'lower'.
+
+    References
+    ----------
+    .. [1] "Quartile Coefficient of Dispersion", *Wikipedia*,
+    https://en.wikipedia.org/wiki/Quartile_coefficient_of_dispersion.
```

Comment:
```suggestion
    .. [1] "Quartile Coefficient of Dispersion", *Wikipedia*,
           https://en.wikipedia.org/wiki/Quartile_coefficient_of_dispersion.
```

Reply:
To fix this, just click "Add suggestion to batch" for this and any suggestions below that you agree with, then "Commit suggestions"